### PR TITLE
Partfive

### PR DESCRIPTION
**Pull request by harringtonben**
@morecallan this biznazz is ready for review.

### CODE REVIEW

**Comment by rousell** [comment]
@harringtonben cool! two things: (1) change your nav bar brand from 'Movie History' (2) it makes me sad that you used a javascript alert. try implementing it as a modal that you have to confirm the deletion. if they dismiss the modal, then the contact won't be blocked. (or you could also implement the alert as a toast message, that would also be acceptable)

**Comment by harringtonben** [comment]
@rousell Navbar updated and toast added!

File: javascripts/controllers/DetailCtrl.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-app.controller("DetailCtrl", function($routeParams, $scope, DatabaseService) {
+app.controller("DetailCtrl", function($location, $routeParams, $scope, $window, ngToast, DatabaseService) {
```
Comment (rousell) [inline javascripts/controllers/DetailCtrl.js]: remove window you don't use it

**Comment by zoeames** [comment]
image
Why does this give me an error? - it points to facebook and says enter valid url

**Comment by harringtonben** [comment]
@zoeames it requires http://

**Comment by zoeames** [comment]
looks good 👍

**Pull request merged**